tests/network/vmi_networking.go

```diff
@@ -185,17 +185,16 @@ var _ = SIGDescribe("[rfe_id:694][crit:medium][vendor:cnv-qe@redhat.com][level:c
 		})
 
 		Context("with propagated IP from a pod", func() {
-
-			DescribeTable("should be able to reach", func(op k8sv1.NodeSelectorOperator, hostNetwork bool) {
-				if op == k8sv1.NodeSelectorOpNotIn {
-					requirements.MultipleNodes()
-				}
+			DescribeTable("should be able to reach", func(hostNetwork bool) {
+				requirements.MultipleNodes()
 
 				namespace := testsuite.GetTestNamespace(nil)
 				if hostNetwork {
 					namespace = testsuite.NamespacePrivileged
 				}
 
+				const testLabel = "tcp-server"
+				inboundVMI := libvmifact.NewCirros(libvmi.WithLabel(testLabel, ""))
```

Comment:
Looks like dead code to me.

Reply:
Because `inboundVMI` is overloaded by the next line and not used.

Reply:
Done